packages/trader/src/Modules/Trading/Components/Form/TradeParams/Turbos/barrier-selector.jsx

```diff
@@ -107,11 +82,10 @@ const BarrierSelector = ({ barrier_1, onChange, setHoveredBarrier, turbos_barrie
             <MobileDialog
                 title={barriers_header_mobile}
                 onClose={toggleBarriersTable}
-                has_content_scroll={true}
+                has_content_scroll={false}
```

Comment:
```suggestion
```
@kate-deriv no need to pass this prop, it will be false by default if you simply omit it👍🏻